cirq/experiments/cross_entropy_benchmarking.py

```diff
@@ -0,0 +1,347 @@
+import numpy as np
+from typing import List, Set, Tuple, Sequence, Dict, Any
+
+from matplotlib import pyplot as plt
+from cirq import devices, ops, circuits, sim
+
+
+class CrossEntropyResult:
+    """Results from a cross-entropy benchmarking (XEB) experiment."""
+
+    def __init__(self, num_cycle_range: Sequence[int],
```

Comment:
Change this pair of vectors into a `Dict[int, float]`, or into a `Sequence[CrossEntropyPair]` where `CrossEntropyPair = NamedTuple(...)`.

Reply:
Done. Changed to a sequence of NamedTuples.